tools/monorepo-utils/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "monorepo-utils",
+	"version": "0.0.1",
+	"description": "WooCommerce Monorepo utility tooling.",
+	"author": "Automattic",
+	"homepage": "https://github.com/woocommerce/woocommerce",
+	"license": "GPLv2",
+	"repository": "woocommerce/woocommerce",
+	"dependencies": {
+		"chalk": "^4.1.2",
+		"commander": "^9.4.0",
+		"dotenv": "^10.0.0",
+		"ts-jest": "^29.1.0",
+		"ts-node": "^10.2.1"
+	},
+	"devDependencies": {
+		"@commander-js/extra-typings": "^0.1.0",
+		"@tsconfig/node16": "^1.0.3",
+		"@types/jest": "^27.4.1",
+		"@types/node": "^16.18.18",
+		"@woocommerce/eslint-plugin": "workspace:*",
+		"eslint": "^8.32.0",
+		"jest": "^29.5.0",
+		"tslib": "^2.3.1",
+		"typescript": "^4.9.5"
+	},
+	"scripts": {
+		"lint": "eslint . --ext .ts",
+		"test": "jest",
+		"utils": "node -r ts-node/register ./src/index.ts"
```

Comment:
To be honest, I would prefer if we didn't use `ts-node` at all. Given that we should be providing our utility as a `bin` script, it seems perfectly reasonable to just provide the build JS script instead. It _does_ require that the package be built before using, but, I don't see a problem with that. We could even make it part of the `install` for this package that it builds it. the end result there would be that after running `pnpm install` on the repo the tool is already ready to use.

I wonder if there's a way it could check for whether or not it was build prior to executing the bin script? It's probably not necessary, we can just `watch` the script's build, but, just food for thought.